src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -636,10 +636,12 @@ const generateTaskRole = (
   };
   // check cpu job
   if (config.taskRoles[taskRole].resourcePerInstance.gpu === 0) {
-    frameworkTaskRole.task.pod.spec.containers[0].env.push({
-      name: 'NVIDIA_VISIBLE_DEVICES',
-      value: 'none',
-    });
```

Comment:
this block is also used in default scheduler

https://github.com/microsoft/pai/blob/be2629950d9e298d9a5929a10118f4bd850095d6/src/rest-server/config/rest-server.yaml#L35

Reply:
maybe set its default value to `NVIDIA_VISIBLE_DEVICES` in default scheduler

Reply:
Done. Remains the same in default scheduler.